src/UI/SummaryListItem.elm

```diff
@@ -0,0 +1,85 @@
+module UI.SummaryListItem exposing (view)
```

Comment:
Do you guys think it's a good name? IMO is not very explicit what this does, but maybe is just me, WYDT? @PedroHLC @renanpvaz

Reply:
not sure how I feel about it, but I can't think of an alternative really

Reply:
Leave as is now then :D if we come up with a better name we  change in the future